example/rest-api/art-institute/art-institute.php

```diff
@@ -156,6 +201,16 @@ function register_aic_block(): void {
 		],
 	]);
 
+	register_remote_data_block( [
+		'title' => 'Art Institute of Chicago Loop',
+		'icon' => 'art',
+		'instructions' => 'This block displays a set amount of artworks based on the provided limit.', 
```

Comment:
I like this! Here's what just this looks like for anyone interested:

![image](https://github.com/user-attachments/assets/7b186629-868c-423a-b612-fc9d03922f90